openmdao/recorders/case.py

```diff
@@ -560,10 +560,10 @@ def list_outputs(self,
             User defined tags that can be used to filter what gets listed. Only outputs with the
             given tags will be listed.
             Default is None, which means there will be no filtering based on tags.
-        includes : iter of str or None
+        includes : str, list of strs, or None
```

Comment:
This is a nitpick, but includes and excludes could be any iterator of strs rather than just a list of strs.